client/my-sites/store/app/store-stats/store-stats-orders-chart/index.js

```diff
@@ -69,9 +81,15 @@ class StoreStatsOrdersChart extends Component {
 
 	render() {
 		const { data, selectedDate, unit, slug } = this.props;
+		const isNewFeatured = config.isEnabled( 'stats/new-main-chart' );
+
+		const classes = classNames( 'store-stats-orders-chart', {
+			'chart-tabs--new-main-chart': isNewFeatured,
+		} );
+
```

Comment:
Could we apply the new conditional class name directly within the `StoreStatsChart` component?

Reply:
I will apply the wrapper with className `stats--new-main-chart` to gate all the modernized styles in the follow-up refactor PR.